script/deployment/MockRewardPool.s.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+/* solhint-disable no-console */
+
+// external
+import { console2 } from "forge-std/console2.sol";
+import { Script } from "forge-std/Script.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+// script
+import { BroadcastManager } from "../utils/BroadcastManager.s.sol";
+import { JsonDeploymentHandler } from "../utils/JsonDeploymentHandler.s.sol";
+// test
+import { MockEvenSplitGroupPool } from "test/mocks/MockEvenSplitGroupPool.sol";
+
+contract MockRewardPool is Script, BroadcastManager, JsonDeploymentHandler {
+    using stdJson for string;
+
+    constructor() JsonDeploymentHandler("main") {}
+
+    /// @dev To use, run the following command (e.g. for Sepolia):
+    /// forge script script/deployment/MockRewardPool.s.sol:MockRewardPool --rpc-url $RPC_URL --broadcast --verify -vvvv
```

Comment:
It would better give example of deploying on public testnet.

Reply:
Resolved in [`f95ddd8`](https://github.com/storyprotocol/protocol-periphery-v1/pull/50/commits/f95ddd8a89f9901ce2f634e9f4388e4d986b8b92) :)